apps/ops/serializers/variable.py

```diff
@@ -58,10 +58,10 @@ def validate(self, attrs):
     def to_representation(self, instance):
         data = super().to_representation(instance)
         if instance.type == FieldType.select:
-            data['extra_args'] = instance.extra_args.get('options', '')
             data['select_default_value'] = instance.default_value
         if instance.type == FieldType.text:
             data['text_default_value'] = instance.default_value
+        data['extra_args'] = instance.extra_args.get('options', '')
         return data
 
     @classmethod
```

Comment:
There are no issues with the code. It is well-written and adheres to Python best practices according to a knowledge cutoff of September of 2021. The author has made some slight adjustments but it's difficult without more information about the context.